### ISSUE TITLE
New MARK rule does not detect "//MARK: - sometext"

### ISSUE THREAD

**Issue by Sega-Zero** [realm/SwiftLint#805]
Latest swiftlint doesn't detect `//MARK: - mark text` as a warning, but it should. Am i right?

**Comment by nikolay-dementiev** [comment]
Hello. 

Why you think that the construction should be marked as warning?

**Comment by Sega-Zero** [comment]
No space after //

**Comment by nikolay-dementiev** [comment]
.. I can't to find any apple's guidelines, in which says, that need space between "//" and "MARK:" ((

Have you some links to share?

**Comment by Sega-Zero** [comment]
take a look at new MARK rule, please. https://github.com/realm/SwiftLint/issues/749

**Comment by nikolay-dementiev** [comment]
That's right. This issue has been resolved. And in repository it is already exists.

Please, try to update your SwiftLint on your device

For example, on my computer, it would be something like this command: "**brew update && brew upgrade `'brew outdated'`**" in Terminal app  (see there http://www.commandlinefu.com/commands/view/4831/update-all-packages-installed-via-homebrew)

**Comment by Sega-Zero** [comment]
Ok, I'll be more specific:
```shell
~ $ swiftlint version
0.12.0
~ $ echo //MARK: mark text | swiftlint lint --use-stdin
<nopath>:1:2: warning: Mark Violation: MARK comment should be in valid format. (mark)
Done linting! Found 1 violation, 0 serious in 1 file.

$ echo // MARK: mark text | swiftlint lint --use-stdin
Done linting! Found 0 violations, 0 serious in 1 file.

$ echo //MARK: - mark text | swiftlint lint --use-stdin
Done linting! Found 0 violations, 0 serious in 1 file.
```

The last line should produce a warning

**Comment by nikolay-dementiev** [comment]
The latest version of SwiftLint, which is stored currently in the repository, provided by your version of the code is described approximately by such rule: "

```
 nonTriggeringExamples: [
            "// MARK: good\n",
            "// MARK: - good\n",
            "// MARK: -\n"
        ]
```

  This variants of behavior is specifically described and located in the the exclusion.
ie .:
`// MARK: - mark the text` - this is good
`// MARK: -mark the text` - this is bad

So, It is no error or bug, I think so. 

Thank you for your patience in explaining the reasons for treatment. & Have a good day )

**Comment by jpsim** [comment]
Sorry it took a while to get to this. I've pushed a fix in #888.
